### DISCUSSION TITLE
Comparison Benchmark

### DISCUSSION THREAD

**Discussion by RakinRkz** [svpcom/wfb-ng#313] [General]
Is there any benchmark or comparison between normal wifi video streaming and wfb-ng? I would like to find out the latency differences.

**Comment by svpcom** [comment]
What is normal video streaming?  wfb-ng is about transfer udp packets. It add ~5ms latency. All other latency (80 - 200ms) is added by  camera, video codecs and monitor

**Reply by RakinRkz** [reply]
By normal video streaming, I meant the ones which are used more commonly with IP cams, such as rtsp.

**Reply by RakinRkz** [reply]
If there isn't any, I was thinking to do some tests and document them.

**Reply by svpcom** [reply]
There is latency_test program in wfb-ng. It measures latency of udp packets via radio link. Usual latency < 5ms.
